x/ecocredit/server/server.go

```diff
@@ -67,7 +67,7 @@ func NewServer(storeKey storetypes.StoreKey,
 	s.marketplaceStore = marketStore
 	s.BaseKeeper = basekeeper.NewKeeper(baseStore, bankKeeper, baseAddr, basketStore, marketStore, authority)
 	s.BasketKeeper = basketkeeper.NewKeeper(basketStore, baseStore, bankKeeper, basketAddr, authority)
-	s.MarketplaceKeeper = marketkeeper.NewKeeper(marketStore, baseStore, bankKeeper, authority)
+	s.MarketplaceKeeper = marketkeeper.NewKeeper(marketStore, baseStore, bankKeeper, authority, ecocredit.ModuleName)
```

Comment:
I believe it should be its own account, it will also make it easier if it evolves into its own module.

Reply:
@JeancarloBarrios see 3376e761aad7078597f07e5cadc930fc1a88a791 and caa606ce839bc2ce8069499b5e15f0c7bb2ca3f3 where:
* a marketplace fee pool module account is set up,
* and I added a new `MsgGovSendFromFeePool` message so that governance can actually manage the fee pool funds

Let me know if this all looks good

Reply:
yeah It looks good to me